src/app/pages/RewardPage/components/FeesEarnedTab/index.tsx

```diff
@@ -91,9 +93,10 @@ export const FeesEarnedTab: React.FC<IFeesEarnedTabProps> = ({
           color={RewardsDetailColor.Yellow}
           title={t(translations.rewardPage.fee.stakingFee)}
           availableAmount={weiTo18(amountToClaim)}
-          totalEarnedAmount={weiTo18(totalRewardsEarned)}
+          totalEarnedAmount={bignumber(totalStakingFees)
+            .add(weiTo18(amountToClaim))
+            .toString()}
```

Comment:
```suggestion
          totalEarnedAmount={weiTo18(
            bignumber(totalStakingFees).add(amountToClaim),
          )}
```

Reply:
You need to use `weiTo18`, otherwise, the tooltips will look like this
<img width="547" alt="image" src="https://user-images.githubusercontent.com/14878713/189851188-ce7c2183-c6e4-4d52-a461-f5c242068d12.png">

Use this approach everywhere pls.

Reply:
added changes in RewardDetails by using AssetValue, pls check your data